### ISSUE TITLE
One question about regularization

### ISSUE THREAD

**Issue by freegyp** [KaimingHe/deep-residual-networks#45]
I am new to deep learning and I am trying to implement ResNet-34 in Tensorflow myself. I want to know if you applied weight decay to trainable weights in EVERY LAYER.
